less/_components/homepage.less

```diff
@@ -249,14 +249,16 @@
     }
 }
 .featured-section-highlights-prefix {
```

Comment:
we should still set the background dark if we are using white text. There are circumstances where images won't load that we should account for.

Reply:
I added a background-color to .parallax-mirror which is the container of the parallax block. If I add a background color for the .featured-section-highlights-prefix, the bg color goes over the parallax block and hides it.